### ISSUE TITLE
How to add confidence to the detected images

### ISSUE THREAD

**Issue by felixgalaxy** [eriklindernoren/PyTorch-YOLOv3#491]
Hi,how to add confidence to the detected image? Thanks

**Comment by praneet195** [comment]
The output of the softmax is the confidence value for each class
